packages/fast-check/documentation/HowItWorks.md

```diff
@@ -227,7 +227,7 @@ miniFc.assert = (property, { seed = Date.now() } = {}) => {
 };
 ```
 
-In previous section, we did not cover the reason why we opted for pure random generators. In property based we want properties to be reproducible no matter the seed, no matter the hardware and no matter the time... But we also want to have independent runs for each iteration in the loop.
+The previous section did not explian the reason for using pure random generators. In property based testing, we want properties to be reproducible no matter the seed, no matter the hardware and no matter the time... But we also want to have independent runs for each iteration in the loop.
```

Comment:
Do you mean 'explain' instead of 'explian'?

Reply:
Whoops, sure do.
```suggestion
The previous section did not explain the reason for using pure random generators. In property based testing, we want properties to be reproducible no matter the seed, no matter the hardware and no matter the time... But we also want to have independent runs for each iteration in the loop.
```